methods.lua

```diff
@@ -699,6 +704,62 @@ GatherLite.p2p = function()
     database = nil;
 end
 
+-- get random nodes and transmit to guild
+GatherLite.send_all = function(guild,party,remote)
+
+    if not guild then 
+        if not party then
+            return;
+        end
+    end
+
+    local database = {};
+    local skills = {};
+    local last_skill;
+
+    last_skill = nil;
+    for i, skill in pairs(GatherLite.spellIDs) do 
+        if last_skill == skill then
+
+        else
+            for k, node in ipairs(GatherLiteGlobalSettings.database[skill]) do
+                table.insert(database, node);
+            end
+        end
+        last_skill=skill;
+    end
+    last_skill = nil;
+
+    for i = 5, 1, -1 do
```

Comment:
Probably this function should not just send 5 random nodes...?

You'd need rate limiting though, sending more than a few hundred characters of message per second will get you kicked from the servers.

Reply:
Yeah Nonsens^^ Should have take a few more coffee...

Thank you for pointing that out

Reply:
mhm maybe adding something like a que to sending it...